docs/root/development/performance/device_conditions.rst

```diff
@@ -0,0 +1,73 @@
+.. _device_conditions:
+
+Analysis of device conditions
+=============================
+
+Results
```

Comment:
Would be good to say up to what commit these results are representative. So that we know what these results are tied to. I mentioned that in the [binary size](https://envoy-mobile.github.io/docs/envoy-mobile/latest/development/performance/binary_size.html#current-status) docs.

Reply:
I can add that